src/applications/disability-benefits/all-claims/content/veteranDetails.jsx

```diff
@@ -14,15 +14,15 @@ const unconnectedVetInfoView = profile => {
   return (
     <div>
       <p className="vads-u-margin-top--0">
-        This is the personal information we have on file for you.
+        This is the personal information we have on file for you sdfsdf.
```

Comment:
??

Reply:
@micahaspyr looks like some random characters got checked in, can clean it up